.github/workflows/autoprerelease.yml

```
@@ -19,8 +19,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
-          ref: 'main'
-
+  ref: refs/pull/${{ github.event.pull_request.number }}/merge
+  fetch-depth: 0  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)